IntegrationWithPharmacies/TenderApi/Service/MedicineTenderOfferService.cs

```diff
@@ -28,16 +28,12 @@ public List<MedicineTenderOffer> GetAll()
             return MedicineTenderOfferRepository.GetAll();
         }
 
-        public MedicineTenderOffer Create(MedicineTenderOfferDto medicineTenderOfferDto)
-        {
-            return MedicineTenderOfferRepository.Create(MedicineTenderOfferAdapter.MedicineTenderOfferDtoToMedicineTenderOffer(medicineTenderOfferDto));
-        }
-
         public void CreateAllMedicineTenderOffers(List<MedicineTenderOffer> medicineTenderOffers)
         {
             foreach (MedicineTenderOffer medicineTenderOffer in medicineTenderOffers)
             {
-                Create(MedicineTenderOfferAdapter.MedicineTenderOfferToMedicineTenderOfferDto(new MedicineTenderOffer(medicineTenderOffer.MedicineName, medicineTenderOffer.Quantity, medicineTenderOffer.AvailableQuantity, medicineTenderOffer.Price, PharmacyTenderOfferRepository.getNextTenderPharmacyOfferId())));
+                medicineTenderOffer.PharmacyTenderOfferId = PharmacyTenderOfferRepository.getNextTenderPharmacyOfferId();
```

Comment:
Correct the method name 'getNextTenderPharmacyOfferId' to start with a capital letter.